src/access/AccessControl.h

```diff
@@ -104,15 +105,15 @@ class AccessControl
 
         Entry() = default;
 
-        Entry(Entry && other) : mDelegate(other.mDelegate) { other.mDelegate = &mDefaultDelegate; }
+        Entry(Entry && other) : mDelegate(other.mDelegate) { other.mDelegate = &*mDefaultDelegate; }
```

Comment:
Maybe Global should have an AsPointer method?

Reply:
This is now `&mDefaultDelegate.get()` which seems fine.